aiida_sssp_workflow/workflows/convergence/engine.py

```diff
@@ -0,0 +1,156 @@
+"""
+Convergence workchain based on aiida-optimize by
+inherit `aiida_optimize.engines._convergence::_ConvergenceImpl`
+"""
+import typing as ty
+import itertools
+
+import numpy as np
+
+from aiida_optimize.engines.base import OptimizationEngineWrapper
+from aiida_optimize.engines._convergence import _ConvergenceImpl
+from aiida_optimize.engines._result_mapping import Result
+
+__all__ = ['ConvergenceEngine']
+
+
+class _ConvergenceTwoFactorImpl(_ConvergenceImpl):
```

Comment:
I think it's better to call this `TwoFactorConvergenceImpl`

Reply:
Sure! changed.